packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/blogpost-collection-header.twig

```diff
@@ -0,0 +1,17 @@
+{% set dark_font_class = blogpostCollectionHeader.dark_font ? "blogpost-collection-header--dark-font" : ""  %}
+
+
+<div class="blogpost-collection-header {{ dark_font_class }}" style="background-color: {{ blogpostCollectionHeader.bg_color }}">
+  <div class="blogpost-collection-header--inner">
+
+    <div class="blogpost-collection-header__content">
+      {% include "@atoms/decorative-link.twig" with blogpostCollectionHeader %}
+    </div>
+
+    {% if blogpostCollectionHeader.image|raw is not empty %}
+      <div class="blogpost-collection-header__image">
+        {{ blogpostCollectionHeader.image|raw }}
```

Comment:
@beto-aveiga Is there a reason we need to use raw here? I don't think this is something we've needed to do with other images in Mayflower.

Reply:
It was a mistake. I just removed it. Thanks!